std-bits/base/src/main/java/org/enso/base/statistics/Statistic.java

```diff
@@ -0,0 +1,17 @@
+package org.enso.base.statistics;
+
+public interface Statistic {
```

Comment:
I suggest renaming this class - it does not represent any arbitrary statistic but a specific kind of a statistic based on moments computation. Maybe `MomentStatistic` or something?